### DISCUSSION TITLE
CRC error with multiple SD cards

### DISCUSSION THREAD

**Discussion by Mika-ncy** [jomjol/AI-on-the-edge-device#2589] [Q&A]
Hi folks !

I need help to understand and fix a CRC error with multiple SD cards.
I tried to install AI on the edge by webconsole and also manualy. 
Each time, when booting I have an CRC error : "SDCARD: Basic R/W check: (E5) Read back, but wrong CRC" so the program can't find wlan.ini and others files...

I tried on : 
- a brand new Verbatim 16GB premium class 10
- a noname like "Mediarange 32GB" cl 10
- even on an old Transflash 512 MB
Each card is functionnal for others devices or utilisations

Cards are formated completely, in fat32, 32ko, with no partition, files placed on the root, on an up to date windows 11 machine.

Everytime, after the first boot attempt files on the SD card have name changed Wlan become wlan.in so it is unreadable for a second try.

The acces point seem to start in the console, but I have no SSID on my other devices.

I don't know what to try next...

Thanks for the help

**Comment by caco3** [comment]
Maybe your ESP-CAM module is bad?
Do you have another one to try it?

**Reply by Mika-ncy** [reply]
Thanks for the reply.
For the moment, I only have this one. Gonna buy one reported as functional in the discussion, if nobody has an other idea for this CRC error
